display_list/display_list_builder.cc

```diff
@@ -1041,9 +1041,9 @@ void DisplayListBuilder::drawPicture(const sk_sp<SkPicture> picture,
                                      const SkMatrix* matrix,
                                      bool render_with_attributes) {
   matrix  //
-      ? Push<DrawSkPictureMatrixOp>(0, 1, std::move(picture), *matrix,
```

Comment:
There were some `std::move`s that were removed on lines 1044, 1046, and 1059. Was that intentional? Just double-checking :)

Reply:
To be honest, I'm not sure what's going on here. Those `std::move`s don't exist in the upstream code:
https://github.com/flutter/engine/blob/3425e21679c41cf86aa2a34de894259888cc3beb/display_list/display_list_builder.cc#L1043-L1046

Reply:
Looks like Dan removed them between when I forked and when I landed and somehow the rebased branch is showing them :/ I'll sync/rebase again and cross my fingers. Maybe a GitHub display bug?

Reply:
Funnily enough: https://twitter.com/mraleph/status/1569267169963171840